.all-contributorsrc

```diff
@@ -0,0 +1,47 @@
+{
+  "projectName": "CityLAB Slides",
```

Comment:
ServiceAgent:innen?

Reply:
Good catch! The name here should be the repo name (I just discovered this and might need to change that in the oder repos as well... :( ). So it is `service-agentinnen`